script/config.js

```diff
@@ -41,7 +44,9 @@ module.exports = {
 }
 
 function getChannel () {
-  if (appMetadata.version.match(/dev/)) {
+  if (process.env.BUILD_DEFINITIONNAME === 'Atom Nightly') {
```

Comment:
This is a VSTS environment variable but I'm planning to change it to something specific to Atom